UI/css/common.css

```diff
@@ -1,7 +1,19 @@
+.padding-0 {
+    padding: 0 !important;
```

Comment:
Do we really need !important ? If not plz don't use it, specially in common since it will be used in different components

Reply:
yes